src/components/ScholarshipForm.jsx

```diff
@@ -186,8 +185,15 @@ function ScholarshipForm() {
           </Button>
         </div>
       </form>
-      {submissionAlert}
     </Container>
   );
 }
+
+ScholarshipForm.propTypes = {
+  setSubmissionAlert: PropTypes.func,
```

Comment:
Should this maybe be required so it doesn't default to `undefined`? Invoking `undefined` throws an exception I believe.